core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/BufferStateForwarder.java

```diff
@@ -26,32 +26,37 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
     }
 
     @Override
+    public void onRepeatModeChanged(@com.google.android.exoplayer2.Player.RepeatMode int repeatMode) {
```

Comment:
This is the retention policy from their code. I thought it was a good idea to keep it because of the `int`.